File: SQL-Sample/demo_mysql_connection.py
import mysql.connector

mydb = mysql.connector.connect(
  host="localhost",
  user="root",
  password="123456",
  database="android_connection_demo"
)

print(mydb)

mycursor = mydb.cursor()

mycursor.execute("SELECT * FROM personnel")

myresult = mycursor.fetchall()

for x in myresult:
  print(x)

File: SQL-Sample/demo_mysql_connection.js
var mysql = require('mysql');

var con = mysql.createConnection({
  host: "localhost",
  user: "root",
  password: "123456",
  database: "android_connection_demo"
});

con.connect(function (err)
{
  if (err) throw err;
  con.query("SELECT * FROM personnel", function (err, result, fields)
  {
    if (err) throw err;
    console.log(result);
  });
});
